### ISSUE TITLE
[Bookworm] Update platform-modules-dell for Bookworm

### ISSUE THREAD

**Issue by saiarcot895** [sonic-net/sonic-buildimage#16735]
#### Description

In the bookworm branch, platform-modules-dell has been disabled from the Broadcom build as it needs changes for Bookworm. This needs to be updated for Bookworm.

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by jeff-yin** [comment]
Please assign to @arunlk-dell @aravindmani-1 @srideepDell 
(I'm not able to designate assignees)

**Comment by saiarcot895** [comment]
@jeff-yin Looks like I can't assign any of them to this task for some reason. They're not showing up in the list of people I can assign to.

**Comment by saiarcot895** [comment]
Fixed by #17003.
